plugins/cricket-scoreboard/cricket_data_fetcher.py

```
@@ -0,0 +1,476 @@
+        except Exception:
+            pass
+
+    # ---- HTTP ------------------------------------------------------------- #
+
+    def _get_json(self, url: str) -> Optional[Dict[str, Any]]:
+        try:
+            resp = self.session.get(url, headers=_HEADERS, timeout=self.request_timeout)
+            if resp.status_code == 200:
+                return resp.json()
+            logger.warning("Cricket API %s returned HTTP %s", url, resp.status_code)
+        except Exception as e:  # pragma: no cover - network failure path
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)